tests/e2e/login/purge.spec.js

```diff
@@ -269,6 +273,8 @@ describe('Purging on login', () => {
         date: result.date
       });
       purgeDate = result.date;
+    }).catch(() => {
+      process.exit(1);
```

Comment:
Rather than catching and exiting can we just return the promise? This goes for the other two changes in this file too...